### PR TITLE
ICMSLST-1452 Creation of task objects during migration

### PR DESCRIPTION
**Pull request by mikeylockwood**

### CODE REVIEW

**Review by MattHolmes123 — APPROVED** [review]
I take it you will revisit this when importing applications in more complex states?
image

**Comment by mikeylockwood** [comment]
> I take it you will revisit this when importing applications in more complex states? image

Yes I will raise tickets for the other task types. It's not possible to determine them yet with the limited data that has been migrated over at this early stage. This initial PR is just a method of being able to create the tasks during the migration process

File: data_migration/models/task.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+from data_migration.models import ImportApplication
+from web.models import Task
+
+
+class TaskBase:
+    TASK_TYPE: str = ""
+    TARGET = Task
+
+    @classmethod
+    def obj_dicts(cls) -> list[Task]:
```
Comment (davidwtbuxton) [inline data_migration/models/task.py:10]: Better name for this method? It doesn't appear to return dicts.

**Review by davidwtbuxton — APPROVED** [review]

File: data_migration/models/task.py (hunk 1 above)
Comment (mikeylockwood) [reply data_migration/models/task.py:10]: Good point, I changed what it does and didn't change the name

**Pull request merged**